KarhooSDK/Network/HttpClient/TokenRefreshingHttpClient/Interactor/KarhooTokenRefreshNeedWorker.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  KarhooTokenRefreshNeedWorker.swift
+//  KarhooSDK
+//
+//  Created by Aleksander Wedrychowski on 05/10/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+protocol TokenRefreshNeedWorker: AnyObject {
```

Comment:
Could we rename this to `TokenRefreshWorker`?

Reply:
But this worker deals only with logic if token needs to be refreshed.

Reply:
If you have any other idea for the name, I am more than happy to use it, though!